HousingFinanceInterimApi.Tests/V1/Infrastructure/DatabaseContext/LoadChargesTests.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using AutoFixture;
+using HousingFinanceInterimApi.V1.Infrastructure;
+using Microsoft.EntityFrameworkCore;
+using Xunit;
+
+
+namespace HousingFinanceInterimApi.Tests.V1.Infrastructure.DatabaseContext;
+
+[System.Diagnostics.CodeAnalysis.SuppressMessage("Security", "CA2100:Review SQL queries for security vulnerabilities", Justification = "This is a test class")]
+
+
+public class LoadChargesTests : IClassFixture<BaseContextTest>
+{
+    private readonly HousingFinanceInterimApi.V1.Infrastructure.DatabaseContext _context;
+    private readonly Fixture _fixture;
+    private readonly List<Action> _cleanups;
+    private readonly Action<string> _executeProcedure;
+
+    public LoadChargesTests(BaseContextTest baseContextTest)
+    {
+        _context = baseContextTest._context;
+        _fixture = baseContextTest._fixture;
+        _cleanups = baseContextTest._cleanups;
+        _executeProcedure = baseContextTest.ExecuteProcedure;
+    }
+
+    private string GeneratePropRef()
+    {
+        // 8 Digit random number that leads with 9 to prevent clashes e.g. "90023871"
+        return string.Concat("9", _fixture.Create<int>().ToString().PadLeft(7, '0').AsSpan(0, 7));
+    }
+
+    [Fact]
+    public void LoadChargesWorks()
```

Comment:
Can this Test method name be more descriptive about what it is testing?
For example: Should_Read_All_Charges_From_ChargesAux_And_Load_Into_Charges()

Or something like that.

See:
https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea